index.bs

```diff
@@ -49,198 +49,207 @@ conversion domain.
 
 # HTML monkeypatches # {#html-monkeypatches}
 
-Rewrite the anchor element to accept the following attributes:
+Rewrite the <{a}> to accept the following attributes:
 
 <pre class="idl">
 partial interface HTMLAnchorElement {
```

Comment:
Not new in this PR, so better to make another PR to fix it, but this only adds to the `HTMLAnchorElement` interface, and not to the allowed content attributes.

Reply:
Ack.